docs/en/latest/practices/enable-authentication-and-restriction.md

```diff
@@ -27,36 +27,124 @@ Consumers are used for the authentication method controlled by Apache APISIX, if
 
 ## Attributes
 
-### Authentication
+### Authentication methods
 
-#### [`keyAuth`](https://apisix.apache.org/docs/apisix/plugins/key-auth/)
+#### `keyAuth`
 
-Consumers add their key either in a header `apikey` to authenticate their requests.
+Consumers add their key either in a header `apikey` to authenticate their requests. For more information about `keyAuth`, please refer to [APISIX jwt-auth](https://apisix.apache.org/docs/apisix/plugins/key-auth/).
+
+<details>

```

Comment:
😂 May I know what's `details` usage here?

Reply:
![3KYX3F0OV7J9)E59TCVYK{5](https://user-images.githubusercontent.com/79972061/168740386-394eafdd-088e-4e76-af6a-9427513cc3fe.png)
![O)KPZ{9BSQK2V 4@}6E8 HG](https://user-images.githubusercontent.com/79972061/168740400-0854532a-3722-44e2-9f4a-04011ded90be.png)

Reply:
Do we have specific reason to use this? If so, could you add it to the style guide on when to use this?

Reply:
My idea is that there are many authentication methods we haven't written in. If we add configuration to each one later, it may appear very bloated. I think the display effect will be better if we use it. Users can choose the configuration content they care about. What do you think?